### PR TITLE
📓 New Blog Post: `GitOps with Kestra: Beyond the Trend`

### PR DESCRIPTION
**Pull request by Ben8t**

### CODE REVIEW

File: content/blogs/2024-02-06-gitops.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+It’s unsurprising to see every part of the company adding Ops as a suffix in job positions: DataOps, SalesOps, CRMOps, RevOps, etc. Mastering operations is all about adding automated processes and peer-reviewing methods. Hence the trends around “Ops”.
+
+While creating such processes is mostly a human policy matter, our tools still should make this as clear as possible and give us good guidelines.
+This is the goal of this blog post regarding Kestra alongside Git.
+
+
+## Kestra: All As Code and All From the UI
+
+One key point about enabling good development and operation processes is the interface. Our vision at Kestra is to let everyone choose their preferences. As Kestra is fully API-based, it’s easy to expose the Kestra system to different interfaces: Kestra User Interface is the most obvious one but users could also use VS Code extension to develop flows, use Terraform provider to interact with Kestra’s API, use the API directly, etc.
+This openness is very important to let everyone find their spot. For both development and operation.
```
Comment (anna-geller) [inline content/blogs/2024-02-06-gitops.md:32]:
```suggestion
This openness is essential to let everyone find their spot — for both development and operation.
```

File: content/blogs/2024-02-06-gitops.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+You might want to use code to define your assets at some maturity level. Having one single source of truth from which we can deploy, review, and edit those assets is critical.
+
+This is why every developer uses Git nowadays.
+That’s why we introduced new Git tasks and capabilities in Kestra.
```
Comment (anna-geller) [inline content/blogs/2024-02-06-gitops.md:37]:
```suggestion
```

I'd cut this part as we don't operate on asset level + duplication of what was said. Ending the paragraph with this sentence "This openness is essential to let everyone find their spot — for both development and operation." makes the paragraph stronger.

File: content/blogs/2024-02-06-gitops.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+
+## Managing orchestration & business logic with Git in Kestra
+
+Globally speaking, the experience of writing business logic should be decoupled from orchestration logic. With evolving needs, tools, and a dynamic business landscape, constantly changing your orchestration and dependencies is not ideal. The absence of a solid architecture becomes an unintended architecture, which is not the most effective.
+
+In Kestra there are several ways to distinct orchestration, defined through Kestra DSL in YAML, and your business logics (using Python, SQL, Java, etc.):
```
Comment (anna-geller) [inline content/blogs/2024-02-06-gitops.md:44]:
```suggestion
There are plenty of benefits to separating your business logic from orchestration logic. Such a decoupled approach keeps your code portable without vendor lock-in and allows you to evolve your business logic without having to worry about orchestration and vice versa. 

The separation of concerns in your application and orchestration layer helps you stay adaptable to evolving business needs, tools, and a dynamic market landscape. Here are some ways of how you can accomplish that in Kestra:
```

File: content/blogs/2024-02-06-gitops.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+
+Implementing GitOps enables a holistic view of the company stack. It gathers everyone at the same table.
+Having a robust, fast, and consistent platform overcomes the challenges posed by hyperspecialization in various domains — be it data orchestration, process, and microservice orchestration, or infrastructure and business automation.
+
+Applying GitOps aims to provide support for projects to be consistent, manageable, and transparent. That’s what Kestra provides too.
+
+Are you already implementing GitOps in your projects? What are the benefits?
+Please share your views and come to [Kestra’s Slack](https://kestra.io/slack) channel.
+
+Also, go through [Kestra's GitHub repository and add a star!](https://github.com/kestra-io/kestra) Follow us on [Twitter](https://twitter.com/kestra_io)
```
Comment (anna-geller) [inline content/blogs/2024-02-06-gitops.md:116]:
```suggestion
At Kestra, we believe that the GitOps paradigm will extend far beyond its current prominence in software. Data already initiates this shift, but Sales, CRM, HR, Revenue, and every part of the company will find themselves in the need for scalability and collaboration.

Implementing GitOps enables a holistic view of the company stack. Having a robust, fast, and consistent platform overcomes the challenges posed by hyperspecialization in various domains — be it data orchestration, process and microservice orchestration, or infrastructure and business automation.

Applying GitOps aims to provide support for projects to be consistent, manageable, and transparent. That’s what Kestra provides, too.

Are you already implementing GitOps in your projects? What are the benefits?
Please share your views in the [Kestra’s Slack](https://kestra.io/slack) channel.

If you want to learn more about Kestra, check out our [documentation](https://kestra.io/docs) or [request a demo](https://kestra.io/demo), and if you like the project, become our next star on [GitHub](https://github.com/kestra-io/kestra).
```

what happened to that last sentence haha

**Review by anna-geller — APPROVED** [review]
small suggestions + remember the title - I'd suggest sth like "GitOps Patterns for Data and Platform Engineers with Kestra"

File: content/blogs/2024-02-06-gitops.md (hunk 4 above)
Comment (Ben8t) [reply content/blogs/2024-02-06-gitops.md]: done ;) (didn't know what happened, too fast copy-past 😅)

File: content/blogs/2024-02-06-gitops.md (hunk 2 above)
Comment (Ben8t) [reply content/blogs/2024-02-06-gitops.md]: done

**Review by MartinRst — APPROVED** [review]

**Pull request merged**